src/components/FilterBar.jsx

```diff
@@ -51,10 +69,13 @@ function FilterBar() {
           items={sortingOptions}
           defaultSelect="deadlineSoon"
           removeNone
+          givenFunction={modifySorting}
         />
       </Grid>
     </Grid>
   );
 }
 
-export default FilterBar;
+FilterBar.propTypes = {
+  passedFunction: PropTypes.func.isRequired,
```

Comment:
What is `passedFunction` supposed to be? From a naming perspective I can't guess from a quick glance what `FilterBar`'s `passedFunction` is supposed to be. If I created some component named `FooBar` what do you think `passedFunction` would do?

Reply:
Punted for #342